src/lib.rs

```diff
@@ -1,8 +1,8 @@
 #![warn(missing_debug_implementations, rust_2018_idioms, unreachable_pub)]
 #![deny(rustdoc::broken_intra_doc_links)]
 #![doc(test(
-    no_crate_inject,
-    attr(deny(warnings, rust_2018_idioms), allow(dead_code, unused_variables))
+no_crate_inject,
+attr(deny(warnings, rust_2018_idioms), allow(dead_code, unused_variables))
 ))]
```

Comment:
This is breaking lints now

<img width="1041" alt="Screenshot 2022-12-14 at 7 30 36 AM" src="https://user-images.githubusercontent.com/21288394/207622782-b242720c-884d-45c6-9d02-98294eb75331.png">